### PR TITLE
Added support for Devil68 Pro keyboard

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Added support for Devil68 Pro keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/devil68_pro/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rgb_matrix.h"
+#include "progmem.h"
+#include "config.h"
+#include "eeprom.h"
```
Comment (drashna) [inline keyboards/devil68_pro/keymaps/default/keymap.c:20]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]
Should remove any includes in the keymap that are not explicitly required.

File: keyboards/devil68_pro/keymaps/default/keymap.c (hunk 1 above)
Comment (HorrorTroll) [reply keyboards/devil68_pro/keymaps/default/keymap.c]: Done, thanks

**Comment by HorrorTroll** [comment]
I'm just update a bit of folder structure, so it should better than before

**Review by drashna — APPROVED** [review]

**Pull request merged**